apps/web/src/components/tx/security/tenderly/__tests__/useSimulation.test.ts

```diff
@@ -14,15 +14,10 @@ const setupFetchStub = (data: any) => () => {
   })
 }
 
+const originalGlobalFetch = global.fetch
 describe('useSimulation()', () => {
-  afterEach(() => {
-    //@ts-ignore
-    global.fetch?.mockClear?.()
-  })
-
   afterAll(() => {
-    // @ts-ignore
-    delete global.fetch
+    global.fetch = originalGlobalFetch
   })
```

Comment:
@iamacook you wrote the original version of those tests. Why were you deleting the global.fetch object? I didn't fully get it. Can you maybe check if those tests make sense with the new fixed jsdom environment. Maybe you did those things because jsdom was broken.